.travis.yml

```diff
@@ -1,12 +1,12 @@
 language: java
 matrix:
   include:
-    - jdk: openjdk8
```

Comment:
Then we should also bump `version.jdk`, right?

Reply:
@ferdinand-swoboda good point. Without this the PR would only about not supporting JDK 8 for building, but indeed my intention was to also raise the baseline. Pushed a commit.